### PR TITLE
Added Environment variables

### PR DESCRIPTION
**Pull request by ferror18**

### CODE REVIEW

**Review by yirano — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -11,12 +11,14 @@
...
     "axios": "^0.19.2",
     "craco-less": "^1.17.0",
+    "json-web-token": "3.2.0",
     "moment": "^2.29.1",
     "moment-timezone": "^0.5.32",
     "plotly.js": "^1.54.6",
     "react": "^16.13.1",
     "react-bootstrap": "^1.4.0",
     "react-dom": "^16.13.1",
+    "react-dotenv": "0.1.2",
```
Comment (Jahteo) [inline package.json:21]: glad you found the react module!

File: package.json [hunk 2]
```diff
@@ -73,6 +76,11 @@
       "last 1 safari version"
     ]
   },
+  "react-dotenv": {
+    "whitelist": [
```
Comment (Jahteo) [inline package.json:80]: what's the whitelist do?

File: public/env.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+window.env = {
+  "BASE_URL": "https://vbb-backend-team-a.herokuapp.com",
```
Comment (Jahteo) [inline public/env.js:2]: You got the new mock backend up! Nice!!!

File: public/index.html [hunk 4]
```diff
@@ -25,6 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Basic SPA</title>
+    <script id="react-dotenv" src="./env.js"></script>
```
Comment (Jahteo) [inline public/index.html:28]: I don't think we're using this page. I think it's one of the scaffolding pages that we'll be deleting later

Comment (Jahteo) [reply public/index.html:28]: ...though I could be wrong. Esp now that I see it has something to do with the build version...

**Review by Jahteo — APPROVED** [review]

**Pull request merged**